analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -98,16 +98,23 @@ private fun String.extractOrganization(): String? =
         line.withoutPrefix(SpdxConstants.ORGANIZATION)
     }.firstOrNull()
 
+/**
+ * Return true if the id is that of an [SpdxExternalDocumentReference] if it has an additional package id attached to it
+ * or not. Otherwise return false.
```

Comment:
How about:

Return whether this string has the format of an [SpdxExternalDocumentReference], with or without an additional package id attached.